src/events/calculate.js

```diff
@@ -0,0 +1,32 @@
+import Big from 'big-js';
+
+const calculate = (numOne, numTwo, calculation) => {
+  const one = Big(numOne);
+  const two = Big(numTwo);
+  if (calculation === '+') {
+    return one.plus(two).toString();
+  }
+  if (calculation === '-') {
+    return one.minus(two).toString();
+  }
+  if (calculation === '*') {
+    return one.times(two).toString();
+  }
+  if (calculation === '/') {
+    try {
+      return one.divide(two).toString();
+    } catch (error) {
+      return 'Not divisible by 0.';
+    }
+  }
+  if (calculation === '%') {
+    try {
+      return one.mod(two).toString;
+    } catch (error) {
+      return 'Module not found, not divisible by 0.';
+    }
```

Comment:
- [ ] When I perform a modulus operation, I get a blank screen instead of the result of the modulus operation. For example, when I do `9%2`, I should be getting `1` but I get a blank screen instead. Kindly fix it. 

_**Hint:**_ It would be nice to use the _**math logic**_ files provided for you and use them as directed. In development, there is no need to reinvent the wheel when the wheel works perfectly. 😄